llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -566,61 +570,77 @@ TableFiles processOneModule(std::unique_ptr<Module> M, bool IsEsimd,
                                       EmitOnlyKernelsAsEntryPoints,
                                       DeviceGlobals);
 
-  StringRef FileSuffix = IsEsimd ? "esimd_" : "";
+  StringRef FileSuffix0 = IsEsimd ? "esimd_" : "";
 
   for (size_t I = 0; I < MSplit->totalSplits(); ++I) {
-    module_split::ModuleDesc ResMDesc = MSplit->nextSplit();
-    Module &ResM = ResMDesc.getModule();
-
-    bool SpecConstsMet = processSpecConstants(ResM);
```

Comment:
this part (lines 575-624) is now nested inside iteration over "double GRF" split w/o modifications (only renaming variables) - new lines 592-643